ui/app/components/clients/dashboard.js

```diff
@@ -92,12 +93,12 @@ export default class Dashboard extends Component {
       let filteredNamespace = this.filterByNamespace(this.selectedNamespace);
       return filteredNamespace.mounts ? this.filterByNamespace(this.selectedNamespace).mounts : null;
     } else {
-      return this.byNamespaceActivity;
+      return this.getActivityResponse.byNamespace;
     }
   }
 
   get responseTimestamp() {
-    return this.args.model.activity?.responseTimestamp;
+    return this.getActivityResponse.responseTimestamp;
```

Comment:
on init, does getActivityResponse have any data? I thought on init, the data would be on this.args.model.activity and then after a queried response either through filter or the date pickers, we would return this.activityResponse. Or maybe I'm reading this wrong.

Reply:
Great question! Everywhere this component relied on `this.args.model.activity` I changed to `getActivityResponse` 
Which returns either the passed in arg `model.activity` (the initial response on init, using license start_date). Then subsequently updates as dates are queried. Happy to walk through if you have questions (and make sure this is a good way to approach it). Here's the getter for reference:

```  // on init API response uses license start_date, getter updates when user queries dates
  get getActivityResponse() {
    return this.queriedActivityResponse || this.args.model.activity;
  }
`

Reply:
ah got it. Thanks for the explanation!